press/press/doctype/usage_record/usage_record.py

```diff
@@ -29,6 +29,9 @@ def on_cancel(self):
 	def update_usage_in_invoice(self):
 		invoice_type = "Summary" if self.prepaid else "Subscription"
 		team = frappe.get_doc("Team", self.team)
```

Comment:
Here you can do something like,

team = frappe.get_doc("Team", self.parent_team or self.team)

instead of if loop below